documentation/modules/managing/proc-drain-cleaner-using.adoc

```diff
@@ -14,6 +14,12 @@ The Cluster Operator performs rolling updates based on the annotation.
 
 * You have xref:proc-drain-cleaner-deploying-{context}[deployed the Strimzi Drain Cleaner].
 
+.Considerations when using anti-affinity configuration 
+
+If using xref:assembly-scheduling-str[anti-affinity] with your Kafka or ZooKeeper pods, consider adding a spare worker node to your cluster or allowing pod rescheduling on the master node.
```

Comment:
We should not recommend rescheduling on master nodes.

Reply:
Yes, that's just a workaround that could be used in dev/test, not in production.

Reply:
Okay. I've take out the master node option.